eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -122,40 +122,40 @@ import Prelude hiding (lookup)
 
 -- ======================================
 
-data TxBodyRaw era = TxBodyRaw
-  { _inputs :: !(Set (TxIn (EraCrypto era))),
-    _collateral :: !(Set (TxIn (EraCrypto era))),
-    _outputs :: !(StrictSeq (Core.TxOut era)),
-    _certs :: !(StrictSeq (DCert (EraCrypto era))),
-    _wdrls :: !(Wdrl (EraCrypto era)),
-    _txfee :: !Coin,
-    _vldt :: !ValidityInterval,
-    _update :: !(StrictMaybe (Update era)),
-    _reqSignerHashes :: Set (KeyHash 'Witness (EraCrypto era)),
-    _mint :: !(MultiAsset (EraCrypto era)),
+data AlonzoTxBodyRaw era = AlonzoTxBodyRaw
+  { atbrInputs :: !(Set (TxIn (EraCrypto era))),
```

Comment:
I find these names a bit difficult to process, because of how alien these abbreviations seem (`atbr`, `btbr`, `smat`) - only because they haven't been used, yet so the don't mean anything at the moment. 

On the other hand, I see there's not too many references to them, so maybe it's not so important. 
Just wondering, have you considered using the era name as a prefix? Would it be too long or maybe conflicting with something else? 
I mean 
```
alonzoInputs :: 
alonzoCollateral :: .. 
....
babbageSpendInputs  :: ... 

etc
```

Reply:
I completely agree with you on this @teodanciu. The abbreviations are rather unpronounciable. I don't think they are likely to conflict with anything else, though the names might get "too long" - this is a subjective presumption. I can change this to using full era names if you think it will be helpful.

Reply:
Maybe let's see what others think about it before changing, maybe someone has a better idea altogether!

Reply:
I use a similar style as Aniket, so I'm used to these kinds of names

Reply:
Same here, I was able to decipher `atbr` for `alonzo tx body raw` just fine. This is a common convention in Haskell, in order to avoid really long names.

The important part is the consistency. This gives us a predictable way of uniquely identifying the type the field works with. The naming convention of using scientists' names in alphabetical order helps us avoid those clashes as well if we use just the first latter (eg, `a` for `Alonzo`, `b` for `Babbage`)


Also with respect to all of the parameterized families, `Tx` , `TxBody`, etc. we should use lenses anyways, so these record field names are mostly an internal implementation detail.